### PR TITLE
DBZ-8103 Add docs for `DecodeLogicalDecodingMessageContent` SMT confi…

### PR DESCRIPTION
**Pull request by rkudryashov**
…g options

### CODE REVIEW

**Comment by jpechane** [comment]
@roldanbob Please take a look. The text is factually correct. Thanks

File: documentation/modules/ROOT/pages/transformations/decode-logical-decoding-message-content.adoc [hunk 1]
```diff
@@ -119,3 +119,22 @@ In the preceding example, the SMT applies the following changes to the original
...
+|===
+|Property
+|Description
+|Type
+|Default
+
+|[[decode-logical-decoding-message-content-fields-mapping]]<<decode-logical-decoding-message-content-fields-null-include, `fields.null.include`>>
+|Defines whether to include fields with null values to the decoded structure
+|boolean
+|false
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/transformations/decode-logical-decoding-message-content.adoc:139]:
```suggestion
[cols="15%a,10%a,10%a,65%a"]
|===
|Property
|Type
|Default
|Description

|[[decode-logical-decoding-message-content-fields-null-include]]<<decode-logical-decoding-message-content-fields-null-include, `fields.null.include`>>
|boolean
|false
|Defines whether to include fields with null values to the decoded structure
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
* Updated the anchor ID for the self-referential link to the `fields.null.include` property. 
* Reconfigured the table to more closely resemble the format of similar tables throughout the docs (i.e., list the default category before the description.
* Minor rewording of description.

**Comment by roldanbob** [comment]
> Applied, thanks

@jpechane Looks like we crossed in the mail. Should I apply my changes to `main` via a separate PR?

**Comment by jpechane** [comment]
@roldanbob Just apply it to the main with proper DBZ ide in the commit message. I was probably a bit too fast today :-)

**Pull request merged**